### PR TITLE
Skeletor 1.0.0

### PR DESCRIPTION
**Pull request by jefferis**
@alexanderbates did you want to make a PR?

### CODE REVIEW

**Comment by alexanderbates** [comment]
> @alexanderbates did you want to make a PR?

Only just now. The neuron healing seems very slow now, and I am yet to look into why. But a solution to that could be a different pull request.

**Comment by dokato** [comment]
Oh damn, after 2 hours of trying to incorporate a new `skeletor` I noticed this PR 🤦 let's say it was a useful exercise, I'll just leave it here for reference: https://github.com/dokato/fafbseg/tree/new_skeletor

**Review by alexanderbates — APPROVED** [review]
Update to the neuron stitching by MST required in order to have this work faster with highly fragmented neurons. Currently, when heal is FALSE, only the largest skeleton is taken

File: R/flywire-skeleton.R [hunk 1]
```diff
@@ -255,15 +281,20 @@ skeletor <- function(segments = NULL,
                      n_rays = 20,
                      projection = c("sphere", "tangents"),
                      fallback = "knn",
+                     waves=2,
+                     step_size=1,
                      sampling_dist=500,
                      cluster_pos = c("median", "center"),
                      shape_weight = 1,
                      sample_weight = 0.1,
+                     inv_dist = 100,
```
Comment (jefferis) [inline R/flywire-skeleton.R:290]: @alexanderbates can you document this new argument?

Comment (alexanderbates) [reply R/flywire-skeleton.R:290]: Ah, misnamed. Fixed

**Pull request merged**